### ISSUE TITLE
hello, i can't scale the globe when i use in h5 

### ISSUE THREAD

**Issue by guirenpei** [chrisrzhou/react-globe#22]
![image](https://user-images.githubusercontent.com/19383783/76181915-d0726700-61fd-11ea-8d94-a714a08bf683.png)
as you see.the globe was  to large to h5 page after rendered.but i can't find the function to avoid it,what should i do.or this Component can't perform with h5 page.

**Comment by chrisrzhou** [comment]
@guirenpei, I think I was able to reproduce and notice this bug as well.  A quick fix is to limit the height.  I think there's a bug where it is using the `height` as a priority instead of the `width`.  I'll see if I can swap this priority around since it's probably more important to fit inside the `width` instead of the `height`.

**Comment by guirenpei** [comment]
@chrisrzhou sorry, i'm not good on the webGL,so can't pull request to you.And thanks for your open source.

**Comment by gmmakqa** [comment]
I think a quick fix for this is to alter the FOV in the setSize function:

this.camera.fov = (height > width) ? (height / width) * CAMERA_FOV : CAMERA_FOV;

This works for me anyhow.

**Comment by chrisrzhou** [comment]
Hey everyone this is now fixed in `v5.0.0`.  The issue was `v4.0.0` did not allow a way to set this on the initialization of the globe, and only when updates to the `*options` props are made.

The feature you are looking for here is the `initialCameraDistanceRadiusScale` in `v5.0.0`.  This prop specifies how far the camera should be placed away from the globe (as a scale factor to the globe's radius).  The example here shows how you can zoom out by setting the value in `initialCameraDistanceRadiusScale`: https://react-globe.netlify.app/usage/globe#initial-settings

For your use case, you can do something like this

```js
<ReactGlobe initialCameraDistanceRadiusScale={10} height="100vh" width="100vw" />
```

Note that `v5.0.0` introduces a couple of breaking changes so check the `changelog.md` or Github release if you want to upgrade to it!  There's a couple of bugfixes, improved/flatter props, improved documentations, better support for `Globe` class, and overall a simplified implementation.
